Main.hs

```diff
@@ -9,30 +9,35 @@ module Main where
 import Control.Monad (unless)
 import qualified Data.List as L
 import Text.Printf (printf)
+import System.Win32 (COORD(xPos))
```

Comment:
What is this?

Reply:
@KlestidP delete this line also, please

Reply:
Can you answer why did you need this import?

Reply:
Dear Alex
I do not know where that import came from. As far as I remember, when I loaded the Main.hs file in VS Code, it was automatically there, as I have only edited the code, not the imports.
Best
Klestid